docs/install.rst

```diff
@@ -96,7 +96,7 @@ Run System Tests
 To check that everything is working as it should, we recommend you run the
```

Comment:
Would it make sense to add a similar section on running the system tests at the bottom of `README.md` in the repo root?

Reply:
done